app/models/person.rb

```diff
@@ -18,6 +18,8 @@ class Person < VersionedModel
 
   has_many :profiles, dependent: :destroy
   has_many :moves, through: :profiles
+  has_many :generic_events, as: :eventable, dependent: :destroy # NB: polymorphic association
```

Comment:
can you add a test for this to maintain the pattern we have for associations on models?

Reply:
Sure.